### PR TITLE
fix: remove code about pack ts col in sql_insert_row

### LINKED ISSUE #938: Do not pack ts col in sql_insert_row
OpenMLDB will not parse ts_dimension in server, so it is no need to pack ts col in sdk

### PR DESCRIPTION
**Pull request by keyu813**
close #938

### CODE REVIEW

File: src/sdk/sql_insert_row.cc [hunk 1]
```diff
@@ -84,13 +81,7 @@ bool SQLInsertRow::Init(int str_length) {
 
 void SQLInsertRow::PackDimension(const std::string& val) { raw_dimensions_[rb_.GetAppendPos()] = val; }
 
-bool SQLInsertRow::PackTs(uint64_t ts) {
```
Comment (dl239) [inline src/sdk/sql_insert_row.cc:87]: delete declaration in header

File: src/sdk/sql_cluster_router.cc [hunk 2]
```diff
@@ -956,11 +956,7 @@ bool SQLClusterRouter::PutRow(uint32_t tid, const std::shared_ptr<SQLInsertRow>&
         return false;
     }
     const auto& dimensions = row->GetDimensions();
-    const auto& ts_dimensions = row->GetTs();
     uint64_t cur_ts = 0;
-    if (ts_dimensions.empty()) {
-        cur_ts = ::baidu::common::timer::get_micros() / 1000;
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc:962]: this line should not delete

File: src/sdk/sql_cluster_router.cc [hunk 3]
```diff
@@ -956,11 +956,8 @@ bool SQLClusterRouter::PutRow(uint32_t tid, const std::shared_ptr<SQLInsertRow>&
...
-    }
+    cur_ts = ::baidu::common::timer::get_micros() / 1000;
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc]: ```uint64_t cur_ts = ::baidu::common::timer::get_micros() / 1000;```

File: src/sdk/sql_cluster_router.cc [hunk 4]
```diff
@@ -971,11 +968,7 @@ bool SQLClusterRouter::PutRow(uint32_t tid, const std::shared_ptr<SQLInsertRow>&
                     DLOG(INFO) << "put data to endpoint " << client->GetEndpoint() << " with dimensions size "
                                << kv.second.size();
                     bool ret = false;
-                    if (ts_dimensions.empty()) {
-                        ret = client->Put(tid, pid, cur_ts, row->GetRow(), kv.second, 1);
-                    } else {
-                        ret = client->Put(tid, pid, kv.second, row->GetTs(), row->GetRow(), 1);
-                    }
+                    ret = client->Put(tid, pid, cur_ts, row->GetRow(), kv.second, 1);
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc]: ```bool ret = client->Put(tid, pid, cur_ts, row->GetRow(), kv.second, 1);```

**Review by aceforeverd — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**